apps/misc/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
 # Create your models here.
+
+class Initiative(models.Model):
+    title = models.CharField(max_length=50)
+    description = models.TextField(max_length=5000)
```

Comment:
5000 seems a very large size for the field as the text needs to be displayed on a small card. 1000 here would be enough.